### ISSUE TITLE
Digital Marin: Mismatched start and end tags (</h3>) for some h2 headings.

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1789]
Some headings of level 2 on our pages have the wrong ending HTML tag. For example on the page https://digitalmarin-ca.proudcity.com/residents there are 3 headings of level 2, but two have `class="widget-title"` and one has `class="proud-widget-title"`. Both the "widget-title" headings start fine with the `<h2>` tag, but they end with the `</h3>` tag. This causes a validation error when I run the page through a validator like https://validator.w3.org/.

Example: `<h2 class="widget-title">About</h3>`

While this isn't causing a big problem because the browsers I've used (Chrome and Firefox) handle it just fine, it should be fixed so that I can better see the other validation errors that could cause issues. (And there are several other validation errors - maybe someone at Proud City could do a source code validation check?)

**Comment by curtismchale** [comment]
@kevindherman this has been updated and we can push it live. We were swapping strings and instead of looking for `</h3>` we were looking for `<h3>` (notice the missing `/`) thus it wasn't finding the closing tag and swapping it out for our desired markup.

**Comment by kevindherman** [comment]
@curtismchale great!
